esti/commit_test.go

```diff
@@ -156,3 +159,23 @@ func TestCommitWithTombstone(t *testing.T) {
 	require.NoErrorf(t, verifyResponse(commitResp.HTTPResponse, commitResp.Body),
 		"failed to commit changes repo %s branch %s", repo, mainBranch)
 }
+
+func TestCommitReadOnlyRepo(t *testing.T) {
```

Comment:
Should we also test with force?  (Can definitely be a TODO and issue, just something to improve so force works...)